js/index.js

```diff
@@ -0,0 +1,17 @@
+import Keyboard from './keyboard/keyboard.js';
+
+const div = document.createElement('div');
+div.className = 'container';
+document.body.append(div);
```

Comment:
It would be better if the variable `div` was more specific. For example, `container`.